### DISCUSSION TITLE
Firefox clipboard permission - getting error

### DISCUSSION THREAD

**Discussion by manoj-mukherjee-maersk** [cypress-io/cypress#22327] [Component Testing]
Timed out retrying after 4000ms: 'clipboard-read' (value of 'name' member of PermissionDescriptor) is not a valid value for enumeration PermissionName.

<img width="1437" alt="Screenshot 2022-06-15 at 8 55 53 AM" src="https://user-images.githubusercontent.com/88817537/173730457-b226c50f-1208-42a8-bb46-e8496375bc0d.png">

```js
cy.window()
      .its("navigator.permissions")
      .invoke("query", { name: "clipboard-read" })
      .its("state")
```

**Comment by mudit94** [comment]
any update on this @bahmutov
